cmd/direktiv/main.go

```diff
@@ -55,6 +56,8 @@ func main() {
 		sidecar.RunApplication()
 	case "flow":
 		flow.RunApplication()
+	case "init":
```

Comment:
Can you explain why dinit was introduced as part of this PR? That seems to have nothing to do with DIR-1190, right?